### PR TITLE
Adding cCOP token

### PR DESCRIPTION
**Pull request by celoorgcolombia**

### CODE REVIEW

File: src/data/mainnet/celo-tokens-info.json [hunk 1]
```diff
@@ -748,5 +748,14 @@
     "isStableCoin": true,
     "minimumAppVersionToSwap": "1.68.0",
     "symbol": "PUSO"
+  },
+  {
+    "address": "0x8A567e2aE79CA692Bd748aB832081C45de4041eA",
```
Comment (MuckT) [inline src/data/mainnet/celo-tokens-info.json:753]:
```suggestion
    "address": "0x8a567e2ae79ca692bd748ab832081c45de4041ea",
```

File: src/data/mainnet/celo-tokens-info.json [hunk 2]
```diff
@@ -748,5 +748,14 @@
...
+    "decimals": 18,
+    "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/cCOP.png",
+    "name": "Celo Colombian Peso",
+    "isStableCoin": true,
+    "minimumAppVersionToSwap": "1.68.0",
```
Comment (MuckT) [inline src/data/mainnet/celo-tokens-info.json:758]:
```suggestion
```

**Review by MuckT — CHANGES_REQUESTED** [review]

**Comment by celoorgcolombia** [comment]
@MuckT we already included the changes you requested. Can you please check again?

**Review by MuckT — APPROVED** [review]

**Comment by celoorgcolombia** [comment]
@MuckT Thanks for merging PR but now how we can add the price feed to be tracked?
Chainlik Data Feed https://data.chain.link/feeds/celo/mainnet/cop-usd#operator-chainlayer

**Pull request merged**